x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java

```diff
@@ -1138,6 +1159,11 @@ private Map<String, Object> getConnectorConfigurationFromSearchResult(ConnectorS
         return (Map<String, Object>) searchResult.getResultMap().get(Connector.CONFIGURATION_FIELD.getPreferredName());
     }
 
+    private boolean getIsDeletedFromSearchResult(ConnectorSearchResult searchResult) {
```

Comment:
```suggestion
    private boolean isConnectorDeleted(ConnectorSearchResult searchResult) {
```

I was a bit confused by the name in the beginning. Would `isConnectorDeleted` make it more obvious?